src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -271,15 +271,17 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
             var semanticModel = await document.GetSemanticModelForNodeAsync(token.Parent, cancellationToken).ConfigureAwait(false);
             var enclosingType = semanticModel.GetEnclosingNamedType(token.SpanStart, cancellationToken);
 
-            var symbols = semanticModel.GetSymbols(token, document.Project.Solution.Workspace, bindLiteralsToUnderlyingType: true, cancellationToken: cancellationToken);
+            var symbols = semanticModel.GetSemanticInfo(token, document.Project.Solution.Workspace, cancellationToken)
+                                       .GetSymbols(includeType: true);
 
             var bindableParent = document.GetLanguageService<ISyntaxFactsService>().GetBindableParent(token);
             var overloads = semanticModel.GetMemberGroup(bindableParent, cancellationToken);
 
             symbols = symbols.Where(IsOk)
                              .Where(s => IsAccessible(s, enclosingType))
                              .Concat(overloads)
-                             .Distinct(SymbolEquivalenceComparer.Instance);
+                             .Distinct(SymbolEquivalenceComparer.Instance)
+                             .ToImmutableArray();
```

Comment:
This seems wasteful, since we just do .Any() and .First() below.

I would expect the enumerators to be lighter-weight than copying the items into an array.  However, this is usually a tiny number of symbols.